source/CardGames.py

```diff
@@ -1,9 +1,8 @@
-
 import random
 import os
 
 cardImages = []
-values = [11, 2, 3, 4, 5, 6, 7, 8, 9, 10, 10, 10, 10]
+values = list(range(1,14))
 suits = ["Spades", "Clubs", "Hearts", "Diamonds"]
```

Comment:
This is old. It needs to stay how it is on the left for blackjack values. Fix this before merging